### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by hankewyczz**

### CODE REVIEW

**Review by Lucas-Dunker — CHANGES_REQUESTED** [review]
Zachar's LinkedIn hyperlink seems to lead to a 404, just wanted to let you know before approving

File: readme.md [hunk 1]
```diff
@@ -34,14 +34,13 @@ SearchNEU was originally created and built in 2017 by [Ryan Hughes](https://gith
...
-- Puneet Johal <a href="https://github.com/puneetjohal"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/puneetjohal/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a>
-- Diana Steelman <a href="https://www.linkedin.com/in/dianasteelman/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://www.dianasteelman.com/"><img src="./public/images/link.png" alt="Website" width="12px"></a>
-- Autumn Fung <a href="https://www.linkedin.com/in/autumnfung/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://autumnfung1.wixsite.com/fungsphotos"><img src="./public/images/link.png" alt="Website" width="12px"></a>
+- Zachar Hankewycz <a href="https://github.com/hankewyczz"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/hankewycz.z/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://zacharhankewycz.com/"><img src="./public/images/link.png" alt="Website" width="12px"></a>
```
Comment (hankewyczz) [inline readme.md]:
```suggestion
- Zachar Hankewycz <a href="https://github.com/hankewyczz"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/hankewycz-z/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://zacharhankewycz.com/"><img src="./public/images/link.png" alt="Website" width="12px"></a>
```

**Review by Lucas-Dunker — APPROVED** [review]

File: readme.md [hunk 2]
```diff
@@ -60,6 +59,13 @@ And a special thank you to the following folks for contributing!
 - [Megan Li](https://github.com/megandouli)
 - [Becca Johnson](https://github.com/beccajohnson17)
 - [Kimin Kim](https://github.com/kiminkim724)
+- [Andrew Dai](https://github.com/andrewydai) <a href="https://www.linkedin.com/in/andrewydai/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://andrewdai.com/"><img src="./public/images/link.png" alt="Website" width="12px"></a>
+- [Simar Chadha](https://github.com/SimChadha)
+- [Adam Koulopoulos](https://github.com/koulopoulos)
+- [Puneet Johal](https://github.com/puneetjohal) <a href="https://www.linkedin.com/in/puneetjohal/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a>
+- [Diana Steelman](https://www.linkedin.com/in/dianasteelman/) <a href="https://www.dianasteelman.com/"><img src="./public/images/link.png" alt="Website" width="12px"></a>
+- [Autumn Fung](https://www.linkedin.com/in/autumnfung/) <a href="https://autumnfung1.wixsite.com/fungsphotos"><img src="./public/images/link.png" alt="Website" width="12px"></a>
+- Tingwei Shi
```
Comment (tiingweii-shii) [inline readme.md]: Can you please add my LinkedIn...? https://www.linkedin.com/in/tingwei-shi/

**Review by tiingweii-shii — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (hankewyczz) [inline readme.md]:
```suggestion
- Tingwei Shi <a href="https://www.linkedin.com/in/tingwei-shi/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a>
```

**Review by tiingweii-shii — COMMENTED** [review]
Looks like the github for Lucas, Ally, Pranav, and Sam is linked to github.com instead of their github profile 👀

**Comment by Lucas-Dunker** [comment]
Same as Tingwei, the links seems to just be github.com

File: readme.md [hunk 3]
```diff
@@ -34,14 +34,13 @@ SearchNEU was originally created and built in 2017 by [Ryan Hughes](https://gith
...
+- Vanessa Peng <a href="https://www.linkedin.com/in/vanessa-peng"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://vanessapeng.com"><img src="./public/images/link.png" alt="Website" width="12px"></a>
+- Ally Chao <a href="https://github.com/"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/ally-chao/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a>
+- Pranav Phadke <a href="https://github.com/"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/pranav-phadke/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a>
+- Sam Lyon <a href="https://github.com/"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/samclyon/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://soulware.us/"><img src="./public/images/link.png" alt="Website" width="12px"></a>
```
Comment (hankewyczz) [inline readme.md]:
```suggestion
- Lucas Dunker <a href="https://github.com/Lucas-Dunker"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/lucasdunker/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a>
- Vanessa Peng <a href="https://www.linkedin.com/in/vanessa-peng"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://vanessapeng.com"><img src="./public/images/link.png" alt="Website" width="12px"></a>
- Ally Chao <a href="https://github.com/allychao"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/ally-chao/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a>
- Pranav Phadke <a href="https://github.com/pranavphadke1"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/pranav-phadke/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a>
- Sam Lyon <a href="https://github.com/soulwa"><img src="./public/images/github.png" alt="Github" width="12px"></a> <a href="https://www.linkedin.com/in/samclyon/"><img src="./public/images/linkedin.png" alt="LinkedIn" width="12px"></a> <a href="https://soulware.us/"><img src="./public/images/link.png" alt="Website" width="12px"></a>
```

**Review by Lucas-Dunker — APPROVED** [review]

**Review by tiingweii-shii — APPROVED** [review]
I approve 🦦

**Review by pranavphadke1 — APPROVED** [review]

**Pull request merged**